dali/util/cufile.h

```diff
@@ -46,6 +46,8 @@ class DLL_PUBLIC CUFileStream : public FileStream {
    * buffer and the offset where it should put the data.
    */
   virtual size_t ReadGPU(uint8_t* buffer, size_t n_bytes, size_t offset = 0) = 0;
+  virtual size_t ReadGPUThread(uint8_t* buffer, size_t n_bytes, \
```

Comment:
```suggestion
  virtual size_t ReadGPUThread(uint8_t* buffer, size_t n_bytes,
```
nitpick: I think it's not needed. Right?